tools/http-fault-injector/Dockerfile

```diff
@@ -13,6 +13,8 @@ RUN dotnet dev-certs https
 EXPOSE 7777
 EXPOSE 7778
 
+ENV ASPNETCORE_URLS=http://+:7777;https://+:7778
```

Comment:
These are already set in `appsettings.json`, but when we tried to run the docker container, it ws using the default ports of 5000 and 5001 instead.  This seems like a reasonable setting if we want to ensure ASP.NET uses these ports.